templates/hardwarelab/partials/device_rentals.html

```diff
@@ -1,28 +1,28 @@
 <div class="ui segment">
```

Comment:
This partial can be merged back into the main template since `.replaceWith()` is no longer being used